java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/CPMapTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.cp;
+
+import com.hazelcast.cp.CPMap;
+import com.hazelcast.simulator.hz.HazelcastTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+import com.hazelcast.simulator.utils.GeneratorUtils;
+
+import java.util.UUID;
+
+/**
+ * Note that these tests are designed to be run in isolation w.r.t. one another.
+ */
+public class CPMapTest extends HazelcastTest {
```

Comment:
Can this test be modified so it has a configurable number of CP groups.

Reply:
I would use a Long as value and not a String.

Also there needs to be configurable number of entries. Having a map with a single entry is not realistic scenario.

Reply:
Check out this PR:

https://github.com/hazelcast/hazelcast-simulator/pull/2183